exercises/concept/little-sisters-vocab/.meta/exemplar.py

```diff
@@ -12,10 +15,10 @@ def add_prefix_un(word):
 
 
 def make_word_groups(vocab_words):
-    """
+    """Take a list containing a prefix at the first index, words as the rest, and prepend each word with it.
 
-    :param vocab_words: list of vocabulary words with a prefix.
-    :return: str of prefix followed by vocabulary words with
+    :param vocab_words: list - of vocabulary words with a prefix.
+    :return: str - of prefix followed by vocabulary words with
```

Comment:
```suggestion
    :return: str - prefix :: vocabulary words with
```

Reply:
This one is hard.  I almost think it should be a direct example.  Something like `prefix :: prefix + word :: prefix + word :: ...`

Reply:
But I am probably overthinking it.